traceapp/vis.go

```diff
@@ -13,6 +14,8 @@ import (
 	_ "sourcegraph.com/sourcegraph/appdash/sqltrace"
 )
 
+var ErrTimelineItemValidation = errors.New("timeline item validation error")
```

Comment:
Thanks for reviewing this one! - I've submitted a [PR](https://github.com/sourcegraph/appdash/pull/107) adding the missing docs.